app/src/main/java/org/dhis2/data/forms/dataentry/FormView.kt

```diff
@@ -823,6 +823,36 @@ class FormView(
         viewModel.loadData()
     }
 
+    fun setConfiguration(
```

Comment:
`setConfiguration` can be internal as it is used in the same packaged and does not need to be exposed.